<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tedit</class>
 <widget class="QMainWindow" name="Tedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>现有轨迹</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="merge_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>融合</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trans">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>偏移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>缩放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vedit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>速度编辑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trecord">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>录制新轨迹</string>
    </property>
   </widget>
   <widget class="QComboBox" name="traj1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="traj2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="t_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>选择轨迹1</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>选择轨迹2</string>
    </property>
   </widget>
   <widget class="QLabel" name="base_label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x:              y:              z:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="basex">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="basey">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="basez">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="t_label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>输入基准点</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="basex_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ts">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>输入偏移量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="basey_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="base_label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x:              y:              z:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="basez_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="const_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>450</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="base_label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  倍数:                       恒定值:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vlabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>380</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>输入速度参数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="multi">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="execute">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tdelete">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>删除轨迹</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trajname">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>输入轨迹名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dyes">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dno">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>no</string>
    </property>
   </widget>
   <zorder>base_label_3</zorder>
   <zorder>base_label</zorder>
   <zorder>base_label_2</zorder>
   <zorder>label</zorder>
   <zorder>list</zorder>
   <zorder>merge_2</zorder>
   <zorder>trans</zorder>
   <zorder>scale</zorder>
   <zorder>vedit</zorder>
   <zorder>trecord</zorder>
   <zorder>traj1</zorder>
   <zorder>traj2</zorder>
   <zorder>t_label</zorder>
   <zorder>t_label_2</zorder>
   <zorder>basex</zorder>
   <zorder>basey</zorder>
   <zorder>basez</zorder>
   <zorder>t_label_3</zorder>
   <zorder>basex_2</zorder>
   <zorder>ts</zorder>
   <zorder>basey_2</zorder>
   <zorder>basez_2</zorder>
   <zorder>const_2</zorder>
   <zorder>vlabel</zorder>
   <zorder>multi</zorder>
   <zorder>execute</zorder>
   <zorder>tdelete</zorder>
   <zorder>trajname</zorder>
   <zorder>label_2</zorder>
   <zorder>dyes</zorder>
   <zorder>dno</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
